Simplenote/src/androidTest/java/com/automattic/simplenote/screenshots/ScreenshotTest.java

```diff
@@ -1,58 +1,527 @@
+        // Type the search query
+        onView(withId(searchViewId)).perform(typeSearchViewText(query));
+        onView(withId(searchViewId)).perform(pressImeActionButton());
+    }
+
+    private void loadSideMenuFromNotesList() {
+        // There is no R.id for the menu drawer button
+        onView(allOf(withContentDescription("Open drawer"))).perform(click());
+    }
+
+    private void loadSettingsFromNotesList() {
+        loadSideMenuFromNotesList();
+        loadSettingsFromSideMenu();
+    }
+
+    private void enableDarkModeFromNotesList() {
+        loadThemeSwitcherFromNotesList();
+        // The options have no id, and I couldn't find a way to access them by their text
+        onView(childAtPosition(withId(R.id.select_dialog_listview), 1)).perform(click());
+    }
+
+    private void loadThemeSwitcherFromNotesList() {
+        loadSettingsFromNotesList();
+        loadThemeSwitcherFromSettings();
+    }
+
+    // Notes Editor Screen
+
+    private void dismissNoteEditor() {
+        onView(withContentDescription("Navigate up")).perform(click());
+    }
+
+    // Search Screen
+
+    private void dismissSearch() {
+        onView(withContentDescription("Collapse")).perform(click());
+    }
+
+    // Side Menu
+
+    private void loadSettingsFromSideMenu() {
+        // Note: I couldn't find a way to get a straight reference to the settings item, so I
+        // was left with this brittle position based matching.
+        ViewInteraction navigationMenuItemView = onView(
+                allOf(childAtPosition(
+                        allOf(withId(R.id.design_navigation_view),
+                                childAtPosition(
+                                        withId(R.id.navigation_view),
+                                        0)),
+                        3),
+                        isDisplayed()));
+        navigationMenuItemView.perform(click());
+    }
+
+    // Settings Screen
+
+    private int themePosition = 6;
+    private int logoutPosition = 14;
+    private int passcodePosition = 11;
+
+    // Note: I couldn't find a way to get a straight reference to the item, so I was left with this
+    // brittle position based matching.
+    private void selectSettingsOption(Integer textId, Integer position) {
+        // Also note: I had to use that withId + hasDescendant because simply using the
+        // recycler view id produced multiple views.
+        //
+        // Inspired by https://stackoverflow.com/a/37247925/809944.
+        onView(
+                allOf(
+                        withId(androidx.preference.R.id.recycler_view),
+                        hasDescendant(withText(textId))
+                )
+        )
+                .perform(RecyclerViewActions.actionOnItemAtPosition(position, click()));
+    }
+
+    private void loadThemeSwitcherFromSettings() {
+        selectSettingsOption(R.string.theme, themePosition);
+    }
+
+    private void disableDarkModeFromSettings() {
```

Comment:
```suggestion
    private void enableLightModeFromSettings() {
```

This was confusing at first since disabling **_Dark_** is the same as enabling **_Light_**.  What do you think about changing the name of this method to mimic the similar `enableDarkModeFromSettings` method?

Reply:
Oh yeah! Totally. Brain fart.